src/Be.Vlaanderen.Basisregisters.Shaperon/DbaseCharacter.cs

```diff
@@ -86,6 +54,24 @@ public bool AcceptsValue(DateTimeOffset? value)
             return true;
```

Comment:
Replace
```
                var formatted = value.Value.ToString(Options.DateTimeOffsetFormat);
                if (formatted.Length > Field.Length.ToInt32())
                    return false;
```

with `return AcceptsValue(value.Value);`

Reply:
Same for `AcceptsValue(DateTime? value)` above

Reply:
Done